pycbc/inference/jump_proposals/__init__.py

```diff
@@ -0,0 +1,69 @@
+# Copyright (C) 2019  Collin Capano
```

Comment:
Can you rename this folder? I don't think it should be two words. Maybe just 'jump' or 'moves' like emcee calls it?

Reply:
Fixed... changed it to `jump`.